tests/gcp/installer/util/test_resource/expected_forseti_conf_server.yaml

```diff
@@ -0,0 +1,244 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is an input file that is used by the Forseti prerequisites setup
+# wizard to generate an actual forseti_conf.yaml. It's better to work off
+# the forseti_conf.yaml.sample than this one if you are editing the config
+# by hand.
+
+global:
+
+    # database
+    db_host: 127.0.0.1
+    db_user: root
+    db_name: forseti_security
+
+    # gsuite
+    groups_service_account_key_file: /home/ubuntu/gsuite_key.json
+    domain_super_admin_email: test@forseti_test_gmail.com
+
+    # notification
+    # Remove or comment out the email properties if you do not want Forseti to
+    # send email notifications.
+    email_recipient: rec@forseti_test_gmail.com
+    email_sender: send@forseti_test_gmail.com
+    sendgrid_api_key: TESTING_API_KEY
+
+    # quota
+    # The pre-populated values are the defaults from GCP.
+    max_admin_api_calls_per_100_seconds: 1500
+    max_appengine_api_calls_per_second: 20
+    max_bigquery_api_calls_per_100_seconds: 17000
+    max_cloudbilling_api_calls_per_60_seconds: 300
+    max_compute_api_calls_per_second: 20
+    max_container_api_calls_per_100_seconds: 1000
+    max_crm_api_calls_per_100_seconds: 400
+    max_iam_api_calls_per_second: 20
+    max_results_admin_api: 500
+    max_sqladmin_api_calls_per_100_seconds: 100
+
+##############################################################################
+
+inventory:
+
+    # Valid values are: debug, info, warning, error
+    loglevel: info
+
+    pipelines:
```

Comment:
The pipelines: config is legacy from 1.0 and not used anywhere in code, please delete it from tests and any other configuration files.

Reply:
Removed the inventory section from the yaml file.